pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -383,7 +384,7 @@ func (m *kubeGenericRuntimeManager) GetPods(all bool) ([]*kubecontainer.Pod, err
 
 		converted, err := m.toKubeContainer(c)
 		if err != nil {
-			klog.V(4).Infof("Convert %s container %v of pod %q failed: %v", m.runtimeName, c, labelledInfo.PodUID, err)
+			klog.V(4).InfoS("Convert container of pod failed", "runtimeName", m.runtimeName, "container", c, "podUID", labelledInfo.PodUID, "err", err)
```

Comment:
```suggestion
			klog.V(4).InfoS("Convert container of pod failed", "runtimeName", m.runtimeName, "containerName", c, "podUID", labelledInfo.PodUID, "err", err)
```

Reply:
here also if we log only container name.

Reply:
Yeah this should be containerName to be consistent with other keys.

Reply:
Oh I see we're logging the whole container?